lib/components/private/Field/Field.tsx

```diff
@@ -124,16 +125,22 @@ export const Field = ({
         />
       ) : null}
 
-      <Box position="relative">
-        <BackgroundProvider value={fieldBackground}>
+      <BackgroundProvider value={fieldBackground}>
```

Comment:
Is this redundant now that we have a Box handling the background?

Reply:
Correct ✅